runtime/src/configs/mod.rs

```diff
@@ -242,6 +244,10 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
     type ControllerOriginConverter = XcmOriginToTransactDispatchOrigin;
     type WeightInfo = (); // Configure based on benchmarking results.
     type PriceForSiblingDelivery = NoPriceForMessageDelivery<ParaId>;
+    // Limit the number of messages and signals a HRML channel can have at most
+    type MaxActiveOutboundChannels = ConstU32<128>;
+    // Limit the number of HRML channels
+    type MaxPageSize = ConstU32<{ 1 << 16 }>;
```

Comment:
Let's configure `MaxPageSize` as in https://github.com/r0gue-io/pop-node/pull/198#issuecomment-2297673149

Reply:
@al3mart For the pop-node case, it is understandable but for the templates, I think we should follow the code in Polkadot SDK templates. What do you think?

Reply:
Just thinking that users launching their local environments with `pop up` and the served network file will get a relay with `hrmpChannelMaxMessageSize: 1,048,576` so maybe providing that higher bound could be leveraged by some users.

Reply:
Makes sense @al3mart, updated!